### ISSUE TITLE
WP affords saving and retrieve progression

### ISSUE THREAD

**Issue by iherman** [w3c/wpub#145]
User agents should provide an affordance that saves the reading progression in the publication and return the user to that location the next time that she opens the publication again.

**Comment by iherman** [comment]
Wendy will make a writeup for this affordance, based on the discussion on 2018-05-31

**Comment by iherman** [comment]
Before closing the issue, though: I tried to find references to this affordance in the [UCR document](https://www.w3.org/TR/pwp-ucr/#req_list)  and... I did not find any! Although it sounds like an obvious use case, we forgot about it back then...

I would propose to:

- Move the examples of @wareid into the UCR draft (@atyposh ?)
- Once the new set is there, remove the examples from the WP draft and add references to the new UCR draft... (I or @mattgarrish can do this)

I think this issue should stay open until these minor things are done...

Cc: @TzviyaSiegman

**Comment by iherman** [comment]
I've just realized: there are duplications with the [separate section on progression](https://w3c.github.io/wpub/#aff-progression). Wouldn't it be better (and more concise) to possibly merge those two?

**Comment by atyposh** [comment]
I agree with @iherman that all examples should become links to the same in UCR. Perhaps affordance writers could document their examples as news issues in https://github.com/w3c/dpub-pwp-ucr/issues for me. That would be great :-)

I have also noticed other duplications throughout the document. I'd like to suggest (maybe in the call today) some reorganization of the affordances section, as follows:

- Presentation (I'm currently reorganizing this and may bring "Switch to publication view") into this section
- Progression (to include "Reading Order", "TOC", and "State")
- Search
- Offlining

Thoughts?
